src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/ocrvalidation/client/model/req/OcrDataField.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.req;
+
+public class OcrDataField {
+
+    public final String name;
+    public final String value;
```

Comment:
Don't these two need `@JsonProperty` annotation?

Reply:
only if you want to set the json property name explicitly